_overviews/FAQ/index.md

```diff
@@ -277,3 +277,34 @@ give the compiler, then it's a compiler bug. Please report it on the
 [Scala 2 bug tracker](https://github.com/scala/bug/issues) or [Scala 3
 bug tracker](https://github.com/lampepfl/dotty/issues), but check
 first if it's a duplicate of an existing ticket.
+
+### I set a setting in sbt but nothing happened. Why?
+
+There could be a lot of reasons.  An extremely common one, that
+almost everyone runs into sooner or later, is that you have a bare
+setting in a multi-project build.
+
+For example, if you add this to your `build.sbt`:
+
+    scalaVersion := "2.13.7"
+
+that's a "bare" setting, and you might expect it to apply build-wide.
+But it doesn't. _It only applies to the root project._
+
+In many cases one should simply write instead:
+
+    ThisBuild / scalaVersion := "2.13.7"
+
+Other possibilities include:
+
+* `Global`
```

Comment:
This isn't a good suggestion.  Global is only for defaults (and some exceptional keys that operate outside of all builds).  If you want to change a key across all the projects in the build use ThisBuild.

But you could suggest a local auto-triggering plugin as an alternative to the `.settings(commonSettings)` spammy option.

Reply:
Okay, I removed the `Global` bullet.

Scala FAQ answers intend to be very brief, and any in-depth information is provided only as links. I think the auto-triggering-plugin thing seems beyond the scope of what's necessary to include, it seems like pretty advanced information to me, and I presume it's covered by the doc page we link to.